### PR TITLE
Unknown chunks auto-identification (padding)

### PR DESCRIPTION
**Pull request by qkaiser**
Detects whether unknown chunks are made of single byte padding.

### CODE REVIEW

**Comment by qkaiser** [comment]
@e3krisztian applied your recommendation, fixed the commit message, and rebased

File: unblob/models.py [hunk 1]
```diff
@@ -147,6 +148,27 @@ def as_report(self, entropy: Optional[EntropyReport]) -> UnknownChunkReport:
...
+
+    Important for manual analysis, and analytical certanity: for example
+    entropy, other chunks inside it, metadata, etc.
+
+    These are not extracted, just logged for information purposes and further analysis,
+    like most common bytes (like \x00 and \xFF), ASCII strings, high entropy, etc.
+    """
+
+    def as_report(self, entropy: Optional[EntropyReport]) -> PaddingChunkReport:
+        return PaddingChunkReport(
```
Comment (e3krisztian) [inline unblob/models.py:163]: Maybe we could reuse `ChunkReport` with `handler_name` `padding`?

Comment (qkaiser) [reply unblob/models.py:163]: Smart !

File: unblob/models.py [hunk 2]
```diff
@@ -147,6 +148,27 @@ def as_report(self, entropy: Optional[EntropyReport]) -> UnknownChunkReport:
...
 
+@attr.define(repr=False)
+class PaddingChunk(Chunk):
+    r"""Gaps between valid chunks or otherwise unknown chunks.
+
+    Important for manual analysis, and analytical certanity: for example
+    entropy, other chunks inside it, metadata, etc.
+
+    These are not extracted, just logged for information purposes and further analysis,
+    like most common bytes (like \x00 and \xFF), ASCII strings, high entropy, etc.
```
[the remark this answers is not in the document]
Comment (qkaiser) [reply unblob/models.py]: Removed.

File: unblob/models.py (hunk 1 above)
Comment (qkaiser) [reply unblob/models.py]: Implemented this change and adapted both `test_chunk_in_chunk_report_structure` and `test_flat_report_structure`.

**Review by e3krisztian — APPROVED** [review]

**Pull request merged**